contracts/PrecedenceCampaignArbitrable.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^0.5.8;
+
+import "@aragon/court/contracts/arbitration/IArbitrable.sol";
+import "@aragon/court/contracts/arbitration/IArbitrator.sol";
+
+
+contract PrecedenceCampaignArbitrable is IArbitrable {
+    //bytes4 public constant ERC165_INTERFACE = ERC165_INTERFACE_ID;
+    //bytes4 public constant ARBITRABLE_INTERFACE = ARBITRABLE_INTERFACE_ID;
+    string public constant ERROR_NOT_ALLOWED = "PCA_NOT_ALLOWED";
+
+    address public owner;
+    IArbitrator public arbitrator;
+
+    modifier only(address _who) {
+        require(msg.sender == _who, ERROR_NOT_ALLOWED);
+        _;
+    }
+
+    constructor (address _owner, IArbitrator _arbitrator) public {
+        owner = _owner;
+        arbitrator = _arbitrator;
+    }
+
+    function createDispute(uint8 _possibleRulings, bytes calldata _metadata) external only(owner) {
+        _createDispute(_possibleRulings, _metadata);
```

Comment:
This function should return a uint256 with the dispute id, right?

Reply:
What for? Is any other contract calling it? But I don't think it hurts either.

Reply:
Might as well, we return it from the inner function :).